community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/state/NeoStoreFileListing.java

```diff
@@ -77,7 +78,7 @@ public NeoStoreFileListing( File storeDir, LabelScanStore labelScanStore, Indexi
 
     private void gatherNonRecordStores( Collection<StoreFileMetadata> files, boolean includeLogs )
     {
-        for ( File file : storeDir.listFiles() )
+        for ( File file : Objects.requireNonNull( storeDir.listFiles()) )
```

Comment:
This is functionally equivalent to the previous code, because `Objects.requireNonNull` just throws NPE if the argument is `null`. I don't think it matters very much, though.